monai/apps/auto3dseg/bundle_gen.py

```diff
@@ -0,0 +1,248 @@
+            )
+        modality = data_src_cfg.get("modality", "ct").lower()
+        spacing = data_cfg["stats_summary#image_stats#spacing#median"]
+        spacing[-1] = -1.0
+
+        intensity_upper_bound = float(data_cfg["stats_summary#image_foreground_stats#intensity#percentile_99_5"])
+        intensity_lower_bound = float(data_cfg["stats_summary#image_foreground_stats#intensity#percentile_00_5"])
+        ct_intensity_xform = {
+            "_target_": "Compose",
+            "transforms": [
+                {
+                    "_target_": "ScaleIntensityRanged",
+                    "keys": "@image_key",
+                    "a_min": intensity_lower_bound,
+                    "a_max": intensity_upper_bound,
+                    "b_min": 0.0,
+                    "b_max": 1.0,
+                    "clip": True,
+                },
+                {"_target_": "CropForegroundd", "keys": ["@image_key", "@label_key"], "source_key": "@image_key"},
+            ],
+        }
+        mr_intensity_transform = {
+            "_target_": "NormalizeIntensityd",
+            "keys": "@image_key",
+            "nonzero": True,
+            "channel_wise": True,
+        }
+        for key in ["transforms_infer", "transforms_train", "transforms_validate"]:
+            for idx, xform in enumerate(self.cfg[f"{key}#transforms"]):
+                if isinstance(xform, dict) and xform.get("_target_", "").startswith("Spacing"):
+                    xform["pixdim"] = deepcopy(spacing)
+                elif isinstance(xform, str) and xform.startswith("PLACEHOLDER_INTENSITY_NORMALIZATION"):
+                    if modality.startswith("ct"):
+                        self.cfg[f"{key}#transforms#{idx}"] = deepcopy(ct_intensity_xform)
+                    else:
+                        self.cfg[f"{key}#transforms#{idx}"] = deepcopy(mr_intensity_transform)
+        # self.algorithm_dir = os.path.join(self.output_path, "segresnet2d")
+        # self.inference_script = "scripts/infer.py"
+
+
+class DintsAlgo(BundleAlgo):
+    def fill_template_config(self, data_stats_filename):
+        print("implement dints template filling method")
+
+
+default_algos = {
+    "segresnet2d": dict(
+        _target_="SegresnetAlgo",
+        template_configs="../algorithms/templates/segresnet2d/configs",
+        scripts_path="../algorithms/templates/segresnet2d/scripts",
+    ),
+    "dints": dict(
+        _target_="DintsAlgo",
+        template_configs="../algorithms/templates/dints/configs",
+        scripts_path="../algorithms/templates/dints/scripts",
+    ),
+}
+
+
+class BundleGen(AlgoGen):
+    """
+    This class generates a set of bundles according to the cross-validation folds, each of them can run independently.
+
+    .. code-block:: bash
+
+        python -m monai.apps.auto3dseg BundleGen generate --data_stats_filename="../algorithms/data_stats.yaml"
+
+    """
+
+    def __init__(self, algos=None, data_stats_filename=None, data_lists_filename=None):
```

Comment:
`data_lists_filename` should be "data_src_cfg_name". The config contains datalist/dataroot/multigpu/name/task and optionally, class_names.